web/packages/teleterm/src/ui/AppUpdater/AutoUpdatesManagement.tsx

```diff
@@ -0,0 +1,344 @@
+        >
+          Unreachable clusters
+        </Alert>
+      )}
+      {content && (
+        <Alert
+          width="100%"
+          mb={0}
+          icon={content.kind === 'neutral' ? Cog : undefined}
+          kind={content.kind}
+          details={
+            hasUnreachableClusters && content.showInlineUnreachableErrors
+              ? unreachableDetailsText
+              : content.description
+          }
+          primaryAction={
+            hasUnreachableClusters &&
+            content.showInlineUnreachableErrors && {
+              content: 'Refresh',
+              onClick: props.onCheckForUpdates,
+            }
+          }
+        >
+          {'title' in content ? content.title : ''}
+        </Alert>
+      )}
+      <ManagingClusterSelector
+        autoUpdatesStatus={status}
+        changeManagingCluster={props.changeManagingCluster}
+        isCheckingForUpdates={props.updateEventKind === 'checking-for-update'}
+        getClusterName={getClusterName}
+        // Resets localIsAutoManaged checkbox.
+        key={JSON.stringify(status)}
+      />
+    </>
+  );
+}
+
+function ManagingClusterSelector({
+  autoUpdatesStatus,
+  isCheckingForUpdates,
+  changeManagingCluster,
+  getClusterName,
+}: {
+  autoUpdatesStatus: AutoUpdatesStatus;
+  isCheckingForUpdates: boolean;
+  changeManagingCluster(clusterUri: RootClusterUri | undefined): void;
+  getClusterName(clusterUri: RootClusterUri): string;
+}) {
+  const isAutoManaged =
+    autoUpdatesStatus.enabled &&
+    autoUpdatesStatus.source === 'highest-compatible';
+  // A local state allows us to unselect the checkbox without choosing any managing cluster.
+  const [localIsAutoManaged, setLocalIsAutoManaged] = useState(isAutoManaged);
+
+  const isMostCompatibleCheckboxDisabled =
+    isCheckingForUpdates || !autoUpdatesStatus.options.highestCompatibleVersion;
+  const disabledClusterSelection = localIsAutoManaged;
+  const options = makeOptions({
+    status: autoUpdatesStatus,
+    getClusterName: getClusterName,
+    disabled: disabledClusterSelection,
+  });
+
+  return (
+    <>
+      {(options.length > 1 || autoUpdatesStatus.options.managingClusterUri) && (
+        <Stack width="100%">
+          <label
+            css={`
+              gap: ${p => p.theme.space[1]}px;
+              display: flex;
+            `}
+          >
+            <CheckboxInput
+              checked={localIsAutoManaged}
+              disabled={isMostCompatibleCheckboxDisabled}
+              onChange={e => {
+                setLocalIsAutoManaged(e.target.checked);
+                if (e.target.checked) {
+                  changeManagingCluster(undefined);
+                }
+              }}
+            />
+            Use the highest compatible version from your clusters
+          </label>
+          <>
+            <Text color={disabledClusterSelection ? 'disabled' : undefined}>
+              Choose which cluster should manage updates:
```

Comment:
This has a greenish tint when disabled compared to disabled form controls.

<img width="468" height="406" alt="green-tint" src="https://github.com/user-attachments/assets/cd2415f2-22f7-4e3d-a7ab-fc5ffb10b3d7" />

Reply:
Oh right, fixed.